### PR TITLE
[PATCH v3] Merge ODP linux-generic v1.40.0.0 into linux-dpdk

### PR DESCRIPTION
**Pull request by MatiasElo**

### CODE REVIEW

**Comment by MatiasElo** [comment]
V2:
- Removed WIP tags

File: platform/linux-dpdk/include/odp_config_internal.h [hunk 1]
```diff
@@ -133,10 +133,10 @@ extern "C" {
 /*
  * Number of shared memory blocks reserved for implementation internal use.
  *
- * Each stash and packet pool requires one SHM block, and 20 blocks are
- * reserved for per ODP module global data.
+ * Each packet pool requires one SHM block and 20 blocks are reserved for per
```
Comment (TuomasTaipale) [inline platform/linux-dpdk/include/odp_config_internal.h:136]: The "per" could dropped.

**Review by TuomasTaipale — APPROVED** [review]
One minor comment, otherwise the set looks OK.

File: platform/linux-dpdk/include/odp_config_internal.h (hunk 1 above)
Comment (MatiasElo) [reply platform/linux-dpdk/include/odp_config_internal.h:136]: Fixed in V3.

**Pull request merged**